R/recipe_run.R

```diff
@@ -53,6 +84,15 @@ orderly_run <- function(name, parameters = NULL, envir = NULL,
 
 ##' @export
 ##' @rdname orderly_run
+##' @examples
+##' # The function orderly_data does all the preparation work that
+##' # orderly_run does, but does not run the report but instead
```

Comment:
_but [...] but [...]_
reads awkwardly
_...but does not run the report, instead it ..._
is maybe better.